en/docs/install-and-setup/setup/api-controller/managing-apis-api-products/managing-apis-and-api-products.md

```diff
@@ -510,4 +510,123 @@ For more information, see [Download and Initialize the apictl]({{base_path}}/ins
+3.  Run the corresponding apictl command below to get the logging details of APIs or an API in an environment.
+
+    1. Get the logging details of APIs or an API in an environment.
+
+        -   **Command**
+            ``` bash
+            apictl get api-logging -e <environment>
+            ```
+            ``` bash
+            apictl get api-logging -e <environment> --tenant-domain <tenant domain> 
+            ```
+            ``` bash
+            apictl get api-logging --environment <environment> --tenant-domain <tenant domain> 
+            ```
+            ``` bash
+            apictl get api-logging  -e <environment> --api-id <api-id> 
+            ```
+            ``` bash
+            apictl get api-logging  -e <environment> --api-id <api-id> --tenant-domain <tenant domain> 
+            ```
+
+            !!! info
+                **Flags:**  
+                
+                -   Required :  
+                    `--environment` or `-e` : The environment that the command is executed on  
+                -   Optional :  
+                    `--tenant-domain` : The tenant domain which you need to get the logging details of APIs or an API. If the the flag is not provided, default carbon.super value will be used when executing the command    
+                    `--api-id` : The UUID of the API which you need to get the logging details
+
+            !!! example
+                ```bash
+                apictl get api-logging -e dev 
+                ```
+                ``` bash
+                apictl get api-logging -e dev --tenant-domain wso2.com
+                ```
+                ``` bash
+                apictl get api-logging --environment dev --tenant-domain wso2.com
+                ```
+                ``` bash
+                apictl get api-logging  -e dev --api-id Bf36ca3a-0332-49ba-abce-e9992228ae06 
+                ```
+                ``` bash
+                apictl get api-logging  -e dev --api-id Bf36ca3a-0332-49ba-abce-e9992228ae06 --tenant-domain wso2.com 
+                ```   
+
+        -   **Response**
+
+            ```go
+            API_ID 	                                     API_CONTEXT               	LOG_LEVEL
+            Bf36ca3a-0332-49ba-abce-e9992228ae06         pizzashack/1.0.0	        FULL
+            Af66fa7a-0546-49ba-adfe-a8976522ea96         sampleapi/1.1.0	        BASIC
+            ```
+
+    !!! Info
+        - The `get api-logging` command can be executed only with a user who has super admin permissions.
+        - Use the tenant-domain flag to get the logging details of the APIs in another tenant other than carbon.super tenant while you are logged in as a super admin user.
+
+## Add/Change/Remove the log level of an API in an environment
+
+Follow the instructions below to add/change/remove the log level of an API in an environment using apictl:
```

Comment:
AFAIU, just "change" is enough instead of "add/change/remove"

Reply:
added all the terms, as change only indicated modifying already existing one. WDYT?

Reply:
Kept this place to indicate more details and modified all the other places to "Set"

Reply:
When we create an API, it gets log-level=OFF by default. We are only changing the log level later. I mean that we cannot add or remove a log level.